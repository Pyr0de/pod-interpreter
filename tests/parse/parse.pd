1+2;
(1+3);
(1+2*3);
true;
false;
"asd"+1;
