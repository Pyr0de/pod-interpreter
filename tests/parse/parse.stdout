ExitCode: 0
(+ 1 2)
(group (+ 1 3) null)
(group (+ 1 (* 2 3)) null)
true
false
(+ asd 1)
